File: web/src/model/victron_controller/dashboard/from_0_2_0_command_set-uint-knob.ts
// @ts-nocheck
import {SetUintKnob as v0_2_0_SetUintKnob} from './v0_2_0/Command'
import {SetUintKnob as dashboard_SetUintKnob} from './Command'

export function convert__command__set_uint_knob__from__0_2_0(from: v0_2_0_SetUintKnob): dashboard_SetUintKnob {
    return new dashboard_SetUintKnob (
        from.knob_name,
        from.value
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_command_set_uint_knob.rs


pub fn convert__command__set_uint_knob__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::command::SetUintKnob) -> crate::victron_controller::dashboard::command::SetUintKnob {
    crate::victron_controller::dashboard::command::SetUintKnob {
        knob_name: from.knob_name.clone(),
        value: from.value.clone(),
    }
}